docs/documentation.yml

```
@@ -6,6 +6,8 @@ toc:
 
   - name: Properties and options
     page: 'properties'
+    description: |
+      Mapbox GL JS's global properties and options that you might want to access while initializing your map or accessing information about its status.
```

Small suggestion, take or leave!

```suggestion
      Mapbox GL JS's global properties and options that you can access while initializing your map or accessing information about its status.
```